apis/Google.Cloud.Firestore.Data/Google.Cloud.Firestore.Data/WriteBatch.cs

```diff
@@ -88,6 +88,34 @@ public WriteBatch Delete(DocumentReference documentReference, Precondition preco
         }
 
         /// <summary>
+        /// Adds an update operation that updates just the specified fields paths in the document, with the corresponding values.
+        /// </summary>
+        /// <param name="documentReference">A document reference indicating the path of the document to update. Must not be null.</param>
+        /// <param name="updates">The updates to perform on the document, keyed by the field path to update. Fields not present in this dictionary are not updated. Must not be null.</param>
+        /// <param name="precondition">Optional precondition for deletion. May be null, which is equivalent to <see cref="Precondition.MustExist"/>.</param>
```

Comment:
I can see where you've been c'n'p'ing from ;)

Reply:
Oops, thanks.